auth/api-client/requirements.txt

```diff
@@ -1,3 +1,7 @@
 google-api-python-client==2.47.0
 google-auth-httplib2==0.1.0
-google-auth==2.6.2
+google-auth==2.11.0
```

Comment:
please update `google-auth` after https://github.com/googleapis/google-auth-library-python/pull/1184 is released